__tests__/auth.test.js

```diff
@@ -1,5 +1,5 @@
 import { newDeveloperData } from './__mocks__/documentData';
-import { createDeveloper, loginDeveloper } from './shared/commands';
+import { createDeveloper, loginDeveloper, logoutDeveloper } from './shared/commands';
 
 describe('login', () => {
   it('should successfully log a developer in', async () => {
```

Comment:
can you create a test where the user is unsuccessful in logging in?

Reply:
Done